src/__tests__/api.test.ts

```diff
@@ -23,6 +23,7 @@ describe('Api', () => {
                 },
               ],
               "XrayEnabled": false,
+              "Visibility": true,
```

Comment:
Shouldn't this be `GLOBAL` or `undefined` ? 
Where does `true` come from?

Reply:
I replaced the test with "PRIVATE"